src/base/faucet/Routes.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+  import { Route } from "svelte-routing";
+  import Index from "@app/base/faucet/Index.svelte";
+  import type { Config } from "@app/config";
+  import Submit from "./Submit.svelte";
+
+  export let config: Config;
+</script>
+
+<Route path="faucet">
+  <Index {config} />
+</Route>
+
+<Route path="faucet/submit">
+  <Submit {config} />
```

Comment:
I think `claim` would be a better name for this route/component.

Reply:
Actually `withdraw`, since that's what you call it in the contract / UI

Reply:
Done